### ISSUE TITLE
Voidcraft Lich not spawning

### ISSUE THREAD

**Issue by ZeBrasa** [Cazadorsniper/PO3#559]
**Pack Version:**.41
**Singleplayer/Lan

**Description:**
I can't seem to find void liches anywhere. Tried to wait on erebus swamp, on overworld swamp with a grass platform and creating a swamp dimension with Rftools.

**Comment by Cazadorsniper** [comment]
They are definitely in the erebus swamp (likely killing each other).

**Comment by ZeBrasa** [comment]
Yeah, I saw them before, but for some reason they are not spawning now. Like, when I need them they stop spawning xD

**Comment by ZeBrasa** [comment]
I find a lot of Abyssalcraft mobs though. And lots of acid from previous liches

**Comment by ZeBrasa** [comment]
I took a look on the biometweaker files, and on the "Submerged Swamp" from erebus, there is not a void lich reference. (on the .json file I mean) Probably not related, but am telling you anyway

**Comment by Cazadorsniper** [comment]
Because its id moved...

**Comment by Cazadorsniper** [comment]
Will be fixed next update.

**Comment by Cazadorsniper** [comment]
for now in the swamp config (within biometweaker) change the forBiomes(67) to forBiomes("erebus:submerged_swamp")

**Comment by ZeBrasa** [comment]
Yeah, that's what I thought. I one time did a custom modpack for myself and had that problem too. Thanks for the attention!

**Comment by ZeBrasa** [comment]
Tried what you said, still not getting liches. Probably a thing on my end, will wait for next update
